### ISSUE TITLE
$axios.$delete body payload is missing

### ISSUE THREAD

**Issue by Bigdragon13th** [nuxt-community/axios-module#419]
`@nuxtjs/axios` version: 5.12.2

Issue:
When trying to call API on `delete` method, the request body is missing.

Example problematic code:
```
$axios.$delete('/store/products', {
    data: {
        [
            {
                sku: '1234',
            },
        ],
    },
}
```

More information:
`@nuxtjs/axios` 5.12.1 don't have this problem

**Comment by snsxn** [comment]
+1

**Comment by drocha87** [comment]
I'm experiencing the same problem, looks like the Content-Type: 'application/json' is removed from headers when sending the request. I tried to intercept the request and set the Content-Type header, but without success.

**Comment by Bigdragon13th** [comment]
Looks like it's the issue from Axios@0.20.0

https://github.com/axios/axios/issues/3220

**Comment by ngajhede** [comment]
There is a PR (https://github.com/axios/axios/pull/3282) ready. Question is when 0.20.1 will be released.

A solution we found for now is using `axios.request({url, data, method: 'delete'});`

**Comment by avrtau** [comment]
> There is a PR ([axios/axios#3282](https://github.com/axios/axios/pull/3282)) ready. Question is when 0.20.1 will be released.
> 
> A solution we found for now is using `axios.request({url, data, method: 'delete'});`

`this.$axios.$request` also works.

**Comment by jasonsaayman** [comment]
Hi,

Axios version 0.21.0 has been released 🎉 please use that and let us know if that solves your issue.

Thanks

**Comment by suruaku** [comment]
> Axios version 0.21.0 has been released tada please use that and let us know if that solves your issue.

The issue is [solved](https://github.com/axios/axios/pull/3282) in 0.21.0 

I made merge request with pkg update #436

**Comment by Bigdragon13th** [comment]
@jasonsaayman I've tested Axios 0.21.0 and it fixed the issue, thank you.
But install @nuxtjs/axios 5.12.2 without Axios still give me Axios 0.20.0, so maybe we need to wait for @suruaku 's PR before closing this issue.

**Comment by suruaku** [comment]
> But install @nuxtjs/axios 5.12.2 without Axios still give me Axios 0.20.0, so maybe we need to wait for @suruaku 's PR before closing this issue.

It is because how semver works. 
See [this post](https://michaelsoolee.com/npm-pre-1-caret-rules/ ) or straight from [documentation](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004)

**Comment by Bigdragon13th** [comment]
@suruaku Yeah, I get that

**Comment by danielalvarenga** [comment]
> > But install @nuxtjs/axios 5.12.2 without Axios still give me Axios 0.20.0, so maybe we need to wait for @suruaku 's PR before closing this issue.
> 
> It is because how semver works.
> See [this post](https://michaelsoolee.com/npm-pre-1-caret-rules/) or straight from [documentation](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004)

Wainting for that! :raised_hands:

**Comment by pi0** [comment]
Merged by #410 and updated with v5.12.3. Sorry for delay.
